### PR TITLE
[Docs] Update `get_started`

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/get_started.md [hunk 1]
```diff
@@ -175,9 +172,9 @@ pip install "mmcv>=2.0.0rc1" -f https://download.openmmlab.com/mmcv/dist/cu116/t
 
 #### 在 Google Colab 中安装
 
-[Google Colab](https://colab.research.google.com/) 通常已经包含了 PyTorch 环境，因此我们只需要安装 MMEngine， MMCV 和 MMDetection 即可，命令如下：
+[Google Colab](https://research.google.com/) 通常已经包含了 PyTorch 环境，因此我们只需要安装 MMEngine，MMCV 和 MMDetection 即可，命令如下：
```
Comment (RangeKing) [inline docs/zh_cn/get_started.md:175]:
```suggestion
[Google Colab](https://colab.research.google.com/) 通常已经包含了 PyTorch 环境，因此我们只需要安装 MMEngine，MMCV 和 MMDetection 即可，命令如下：
```

File: docs/zh_cn/get_started.md [hunk 2]
```diff
@@ -60,7 +62,7 @@ pip install -v -e .
 # "-e" 表示在可编辑模式下安装项目，因此对代码所做的任何本地修改都会生效，从而无需重新安装。
 ```
 
-方案 2. 如果你将 mmdet 作为依赖或第三方 Python 包，使用 MIM 安装
+安装 b：如果你将 mmdet 作为依赖或第三方 Python 包，使用 MIM 安装：
```
Comment (RangeKing) [inline docs/zh_cn/get_started.md:65]:
```suggestion
方案 b：如果你将 mmdet 作为依赖或第三方 Python 包，使用 MIM 安装：
```

File: docs/zh_cn/get_started.md [hunk 3]
```diff
@@ -76,19 +78,19 @@ mim install "mmdet>=3.0.0rc0"
...
 ```
 
-载将需要几秒钟或更长时间，这取决于你的网络环境。完成后，你会在当前文件夹中发现两个文件 `yolov3_mobilenetv2_8xb24-320-300e_coco.py` and `yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth`
+下载将需要几秒钟或更长时间，这取决于你的网络环境。完成后，你会在当前文件夹中发现两个文件 `yolov3_mobilenetv2_8xb24-320-300e_coco.py` 和 `yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth`。
 
-**步骤 2.** 推理验证
+**步骤 2.** 推理验证。
 
-方案 1. 如果你通过源码安装的 MMDetection，那么直接运行以下命令进行验证：
+选项 (a). 如果你通过源码安装的 MMDetection，那么直接运行以下命令进行验证：
```
Comment (RangeKing) [inline docs/zh_cn/get_started.md:85]: `选项 (a).` 和 前面的 `方案 a：` 保持统一，包括加不加括号，`.`和`：`也统一只用一种。

File: docs/zh_cn/get_started.md [hunk 4]
```diff
@@ -223,15 +220,13 @@ docker run --gpus all --shm-size=8g -it -v {DATA_DIR}:/mmdetection/data mmdetect
 
 ### 排除故障
 
-如果你在安装过程中遇到一些问题，请先查看 [FAQ](notes/faq.md) 页面。
-
-如果没有找到解决方案，你也可以在 GitHub 上 [打开一个问题](https://github.com/open-mmlab/mmdetection/issues/new/choose) 。
+如果你在安装过程中遇到一些问题，请先查看 [FAQ](notes/faq.md) 页面。如果没有找到解决方案，你也可以在 GitHub 上[打开一个问题](https://github.com/open-mmlab/mmdetection/issues/new/choose)。
```
Comment (RangeKing) [inline docs/zh_cn/get_started.md:223]:
```suggestion
如果你在安装过程中遇到一些问题，请先查看 [FAQ](notes/faq.md) 页面。如果没有找到解决方案，你也可以在 GitHub 上[提出一个问题](https://github.com/open-mmlab/mmdetection/issues/new/choose)。
```

**Review by RangeKing — APPROVED** [review]

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**